app/views/course/courses/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= div_for(current_course) do
+= div_for(current_course, class: 'page-header') do
```

Comment:
What is the purpose of this `page-header` class?

Reply:
Pushes the course title a bit lower down so it is not blocked by the chevron when the sidebar is hidden.